moveit_ros/planning/planning_request_adapter_plugins/src/fix_start_state_path_constraints.cpp

```diff
@@ -119,19 +120,22 @@ class FixStartStatePathConstraints : public planning_request_adapter::PlanningRe
           return true;
         }
         else
+        {
```

Comment:
This block is redundant with the other error case below. Maybe move them after the if/else conditions so that it's used as the last default case.